packages/graphback/src/loadPlugins.ts

```diff
@@ -1,23 +1,28 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 import { GraphbackPlugin } from "@graphback/core";
 
-export function loadPlugins(pluginConfig: any): GraphbackPlugin[] {
-  if (!pluginConfig) {
+export function loadPlugins(pluginConfigMap: any): GraphbackPlugin[] {
+  if (!pluginConfigMap) {
     return [];
   }
 
   const pluginInstances = [];
-  for (const pluginLabel of Object.keys(pluginConfig)) {
+  for (const pluginLabel of Object.keys(pluginConfigMap)) {
     let pluginName = pluginLabel;
     if (pluginLabel.startsWith('graphback-')) {
       // Internal graphback plugins needs rename
       pluginName = pluginLabel.replace('graphback-', '@graphback/codegen-');
     }
+    // override package name
+    if(pluginConfigMap[pluginLabel].packageNameOverride){
+      pluginName = pluginConfigMap[pluginLabel].packageNameOverride;
```

Comment:
```suggestion
      pluginName = pluginConfigMap[pluginLabel].packageName
```

Reply:
Updated now in https://github.com/aerogear/graphback/pull/2077/commits/0c2dcd6dea81a50ecb5cf339cf535a1999c15680